### PR TITLE
Update client to include refund request functionality

### PR DESCRIPTION
**Pull request by ioannis-kody**
## **Associated JIRA tasks**

PBBV0-157

## **What the code does.**

Add refund functionality

## **Does this code change break backwards compatibility?.**

Describe where and how this code change breaks backwards compatibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,6 +283,55 @@ class CancelResponse:
...
+@dataclass
+class RefundRequest:
+    store_id: str
+    order_id: str
+    amount: float
+```
+
+Request parameters:
+- `store_id` - the ID of your assigned store
+- `order_id` - Unique order ID generated by Kody
```
Comment (kody-joao) [inline README.md:299]: Are you sure this is generated by Kody? It seems to be provided by client, we generate the payment_id

File: README.md [hunk 2]
```diff
@@ -283,6 +283,55 @@ class CancelResponse:
...
+class RefundStatus(Enum):
+    PENDING = 0
+    REQUESTED = 1
+    FAILED = 2
+    UNRECOGNIZED = -1
+
+@dataclass
+class RefundResponse:
+    refund_status: RefundStatus
+    order_id: str  # Unique order ID generated by Kody
```
Comment (kody-joao) [inline README.md:315]: same as above

File: README.md (hunk 1 above)
Comment (ioannis-kody) [reply README.md:299]: From what I see in the code, yes, during payment creation for the first time we have this: 

`orderId = request.orderId ?: UUID.randomUUID()`

Comment (ioannis-kody) [reply README.md:299]: Ah, I see what you mean, now !

**Review by kody-joao — APPROVED** [review]

**Pull request merged**